### DISCUSSION TITLE
PayPal one-time donation

### DISCUSSION THREAD

**Discussion by coaster007** [opencollective/opencollective#4917] [Q&A]
i'am missing one-time donations with PayPal.
will that be possible in the future?

**Comment by Betree** [comment]
@coaster007 Could you expand on this? What's your collective, where are you trying to contribute?

Our PayPal implementation supports both one-time and recurring.

**Reply by coaster007** [reply]
Attached the screencapture, no PayPal option.

![image](https://user-images.githubusercontent.com/34204590/141459323-7cfeb46b-81ea-4625-a0ff-f08802d54c2d.png)

**Reply by SudharakaP** [reply]
Out [Paypal implementation is in Beta](https://docs.opencollective.com/help/fiscal-hosts/fiscal-host-settings/connected-accounts#receiving-donations-through-paypal) and only a selected number of hosts have chosen to enable it. The host in question here (Open Collective Europe) has not enabled this option which is the reason you don't see it. 🙂 

cc: @Betree

**Reply by Betree** [reply]
Correct. 

@coaster007 I would recommend contacting your fiscal host (OC Europe) to ask them if they're ready to enable PayPal. We'll be happy to help them set it up if they're willing to do it.
